### DISCUSSION TITLE
Welcome to fds Discussions!

### DISCUSSION THREAD

**Discussion by x1ah** [x1ah/fds#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ushuz** [comment]
可以试试用 Google Sheets + Apps Script 自定义函数，来罗列资产配置
